components/playgroundEditor/index.tsx

```diff
@@ -9,17 +9,26 @@ import React, {
   useRef,
   useMemo,
 } from "react";
-import tryLoadPyodide from "lib/pyodide";
 import useTranslation from "hooks/translation";
 import PlayArrow from "@material-ui/icons/PlayArrow";
-import checkWasm from "lib/wasm";
 import { useDarkTheme } from "hooks/darkTheme";
 import { XTerm } from "xterm-for-react";
 import { FitAddon } from "xterm-addon-fit";
+import CodeRunner from "lib/playground/codeRunner";
+import PythonCodeRunner from "lib/playground/pythonCodeRunner";
+import PistonCodeRunner from "lib/playground/pistonCodeRunner";
 import classes from "./style.module.css";
 
-let executeCode: (code: string) => void;
-let loading = false;
+function getMonacoLanguageName(language: string) {
+  switch (language) {
+    case "c-plus-plus":
+      return "cpp";
+    case "c-sharp":
+      return "cs";
```

Comment:
Python not supported anymore? 🤔

Reply:
It still is, those are just aliases for the text editor. What we call `c-plus-plus`, the editor just recognizes as `cpp`. `python` is `python` and thus doesn't need an alias.